addOns/ascanrulesAlpha/src/main/javahelp/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html

```diff
@@ -102,5 +102,9 @@ <H2>Trace.axd Information Leak</H2>
 <H2>User Agent Fuzzer</H2>
 This active scanner checks for differences in response based on fuzzed User Agent (eg. mobile sites, access as a Search Engine Crawler). The scanner compares the response statuscode and the hashcode of the response body with the original response.
 
+<H2>XSLT Injection Scanner</H2>
+This active scanner checks for certain responses induced by injecting XSL transformations. <br/>
+When injecting tokens produces errors suggesting a vulnerability, the scanner attempts to exploit it by checking the responses to requests trying to access sensitive system information.
```

Comment:
This isn't quite right anymore after the changes. It still tests error conditions but further tests are no longer dependent upon it. So how about something like:

> Attempts various payloads which may induce: error responses, disclosure of library/framework vendor name, remote port scanning, or command execution.